localization/pose_initializer/config/pose_initializer.yaml

```diff
@@ -0,0 +1,165 @@
+/**:
```

Comment:
We use `*.param.yaml` as naming convention for ros parameter yaml file, so `pose_initializer.param.yaml` is better for this file name.

Reply:
Also, please revert the change done by `pre-commit-ci`!